armi/reactor/grids/hexagonal.py

```diff
@@ -572,3 +575,45 @@ def generateSortedHexLocationList(self, nLocs: int):
         )
 
         return locList[:nLocs]
+
+    def rotateIndex(self, loc: IndexLocation, rotations: int) -> IndexLocation:
+        """Find the new location of an index after some number of CCW rotations.
+
+        Parameters
+        ----------
+        loc : IndexLocation
+            Starting index
+        rotations : int
+            Number of counter clockwise rotations
+
+        Returns
+        -------
+        IndexLocation
+            Index in the grid after rotation
+
+        Notes
+        -----
+        Rotation uses a three-dimensional index in what can be known elsewhere
+        by the confusing name of "cubic" coordinate system for a hexagon. Cubic stems
+        from the notion of using three dimensions, ``(q, r, s)`` to describe a point
+        in the hexagonal grid. The conversion from ARMI's indexing follows::
```

Comment:
So, I need to go through ARMI docs and docstrings and fix this globally, but just FYI:

We can not use "ARMI" as a noun, any more. The lawyers argue that this can lead to "genericification" and can cause us to lose our trademark.

So, we are compelled to say things like "the ARMI framework" or "the ARMI data model" or "the ARMI software" or "the ARMI database", etc.

Sorry!  Not my call.